packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_screen.dart

```diff
@@ -52,119 +62,251 @@ class _DeepLinkPageState extends State<DeepLinkPage>
 
   @override
   Widget build(BuildContext context) {
-    return ListView(
-      children: [
-        Row(
-          children: [
-            Expanded(
-              child: Text(
-                'All deep links',
-                style: Theme.of(context).textTheme.bodyLarge,
+    return DefaultTabController(
+      length: 3,
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Row(
```

Comment:
Can you try to use `AreaPaneHeader` for consistent styling with the rest of DevTools?